api/envoy/type/matcher/regex_match_and_subst.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package envoy.type.matcher;
+
+import "envoy/type/matcher/regex.proto";
+
+option java_package = "io.envoyproxy.envoy.type.matcher";
+option java_outer_classname = "RegexMatchAndSubstProto";
+option java_multiple_files = true;
+
+// [#protodoc-title: Regex match and substitute]
+
+// Describes how to match a string and then produce a new string using a regular
+// expression and a substitution string.
+message RegexMatchAndSubstitute {
```

Comment:
Just have this message in `regex.proto` rather than a new file. I don't think this need a new proto file.

Reply:
I've made the requested change.  The `RegexMatchAndSubstitute` message is now in the existing `regex.proto` rather than in a new file.